kobo/apps/hook/tests/test_email.py

```diff
@@ -34,7 +34,7 @@ def _create_periodic_task(self):
     def test_notifications(self):
         self._create_periodic_task()
         first_log_response = self._send_and_fail()
-        failures_reports.delay()
+        failures_reports.apply_async(queue='kpi_low_priority_queue')
```

Comment:
It's too bad that we have to specify the queue when calling the task instead of in the task definition itself, but I don't see any way to do the latter. I found https://stackoverflow.com/questions/64606915/specify-queue-for-a-task-class-in-celery, but it looks like what they're doing there is completely undocumented.

Reply:
I did not want to change the code a lot for this urgent matter but we could replace the `shared_task` decorator with the `celery.app` decorator and use the `queue` argument, e.g:

```
@shared_task
def failures_reports():
    ...
    
    failures_reports.apply_async(queue='kpi_low_priority_queue')
```

with 

```
from kobo.celery import celery_app

@celery_app.task(queue='kpi_low_priority_queue')
def failures_reports():
    ...
    failures_reports.delay()
```

or use maybe some [manual routing](https://docs.celeryq.dev/en/stable/userguide/routing.html#manual-routing)?

Reply:
Is `@celery_app.task(queue='kpi_low_priority_queue')` documented? It's probably staring me in the face :flushed: